ci/scripts/swift_test.sh

```diff
@@ -19,13 +19,29 @@
 
 set -ex
 
+cp ${1}/swift/swiftlint/.swiftlint.yml ${1}/swift/Arrow
```

Comment:
@abandy How about simplifying this? (Can I push this to this branch?)

```diff
diff --git a/ci/scripts/swift_test.sh b/ci/scripts/swift_test.sh
index b6cff9819..b523e3891 100755
--- a/ci/scripts/swift_test.sh
+++ b/ci/scripts/swift_test.sh
@@ -19,9 +19,6 @@
 
 set -ex
 
-cp ${1}/swift/swiftlint/.swiftlint.yml ${1}/swift/Arrow
-cp ${1}/swift/swiftlint/.swiftlint.yml ${1}/swift/ArrowFlight
-
 data_gen_dir=${1}/swift/data-generator/swift-datagen
 export GOPATH=/
 pushd ${data_gen_dir}
@@ -30,16 +27,9 @@ go run .
 cp *.arrow ../../Arrow
 popd
 
-source_dir=${1}/swift/Arrow
+source_dir=${1}/swift
 pushd ${source_dir}
-swiftlint --strict ${1}/Sources/Arrow/*.swift
-swiftlint --strict ${1}/Tests/ArrowTests/*.swift
-popd
-
-source_dir=${1}/swift/ArrowFlight
-pushd ${source_dir}
-swiftlint --strict ${1}/Sources/ArrowFlight/*.swift
-swiftlint --strict ${1}/Tests/ArrowFlightTests/*.swift
+swiftlint --strict
 popd
 
 source_dir=${1}/swift/Arrow
diff --git a/swift/swiftlint/.swiftlint.yml b/swift/.swiftlint.yml
similarity index 65%
rename from swift/swiftlint/.swiftlint.yml
rename to swift/.swiftlint.yml
index 89e8d7bb9..9ce377285 100644
--- a/swift/swiftlint/.swiftlint.yml
+++ b/swift/.swiftlint.yml
@@ -16,15 +16,18 @@
 # under the License.
 
 included:
-  - Sources
+  - Arrow/Sources
+  - Arrow/Tests
+  - ArrowFlight/Sources
+  - ArrowFlight/Tests
 excluded:
-  - Sources/Arrow/File_generated.swift
-  - Sources/Arrow/Message_generated.swift
-  - Sources/Arrow/Schema_generated.swift
-  - Sources/Arrow/SparseTensor_generated.swift  
-  - Sources/Arrow/Tensor_generated.swift
-  - Sources/ArrowFlight/Flight.grpc.swift
-  - Sources/ArrowFlight/Flight.pb.swift
+  - Arrow/Sources/Arrow/File_generated.swift
+  - Arrow/Sources/Arrow/Message_generated.swift
+  - Arrow/Sources/Arrow/Schema_generated.swift
+  - Arrow/Sources/Arrow/SparseTensor_generated.swift
+  - Arrow/Sources/Arrow/Tensor_generated.swift
+  - ArrowFlight/Sources/ArrowFlight/Flight.grpc.swift
+  - ArrowFlight/Sources/ArrowFlight/Flight.pb.swift
 identifier_name:
   min_length: 2 # only warning
-allow_zero_lintable_files: false
\ No newline at end of file
+allow_zero_lintable_files: false
```

Reply:
@kou please do, that would be great!  Thank you!

Reply:
Done!